lib/msal-common/package.json

```diff
@@ -63,8 +63,8 @@
     "jest": "^27.0.4",
     "lodash": "^4.17.21",
     "rimraf": "^3.0.2",
-    "rollup": "^2.46.0",
-    "rollup-plugin-typescript2": "^0.29.0",
+    "rollup": "^3.14.0",
```

Comment:
Looks like this requires Node 14+, which becomes a problem for our Node 10/12 pipelines

Reply:
Good a forcing function for our discussion about what the plan is for common :-)

Reply:
@sameerag - What's the plan for Node vNext?  @hectormmg - FYI

Reply:
Are those pipelines checks for PRs?  I didn't see a link to those.

Reply:
Hmm interesting...looks like [they were disabled](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/f4678bff7332890e0f3e680244a9f6557ef6015b/.github/workflows/msal-node-e2e.yml#L126) for some reason...@hectormmg any idea why?

Reply:
Looks like this: Some tooling no longer supports old versions of node - this job will downgrade certain dependencies for testing

Reply:
@shoatman msal-node v2 looks like in a quarter as per the plan (provided we have resourcing - min case we can drop old node versions). Let's take this offline, cc @bgavrilMS

Reply:
@tnorling I vaguely remember us making the decision to temporarily disable them because they were breaking without a clear cause. We just never came back around to it until now.

@shoatman I haven't had a Node v2 discussion with anyone. I'm not strictly opposed to releasing Node at the same time as the browser libraries, granting that we'll lag behind (or skip ahead) the Node.js major version release schedule by 6mos.

Reply:
@tnorling https://github.com/AzureAD/microsoft-authentication-library-for-js/actions/runs/4127019461/jobs/7129701390

Lerna started throwing "Unexpected token . " on Node 10 and 12, which seems like a breaking change in lerna.

Reply:
Per this thread https://github.com/lerna/lerna/issues/3368 lerna doesn't support either Node 10 nor 12 (as expected, honestly)

Reply:
So the plan is to leave it as is and accelerate dropping these for msal-node?

Reply:
@sameerag I'm not sure. We will eventually move to workspaces and drop lerna (I believe) but dropping unsupported versions of Node ASAP also has to happen. I don't see a way to keep 10 and 12 in our CI/CD without a lerna fix.